### DISCUSSION TITLE
Great webinar today- learned a few things

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1444] [Q&A] [answered]
Thanks for the webinar today was well worth the time. 

What is the ETA for a development kit ,SDK or whatever so we can start working on themes and blocks for Vanjaro?

**Comment by meetmandeep** [comment, marked as answer]
Our focus right now is to get the product released. After official release, we'll start working on tutorials, documentation, and webinars for development.
